### PR TITLE
Added more animal sound effects needed for pettable animals and gibbed head fix

### PR DESCRIPTION
**Pull request by CrzyPotato**
## About the PR 

**Screenshots**

https://user-images.githubusercontent.com/75244093/163080972-b3c70ecd-a65b-4e2d-820d-870778745bbb.mp4

**Changelog**

:cl:
- add: Added more animal sounds for pettable animals! Some of these animals include sloths, foxes, lizards, cows, goats, and more.
- add: fixed issue with gibbed head being able to change direction via clicking

### CODE REVIEW

File: Resources/Audio/Animals/license.txt [hunk 1]
```diff
@@ -12,4 +12,27 @@ chicken_cluck_happy.ogg: modified from "Chicken Single Alarm Call" by freesound
...
+The following sounds were used from youtube:
+
+penguin_squawk.ogg: audio is recorded/provided by youtube user "ProSounds" (https://youtu.be/Anr35RbBL3Y) audio is free to use and downloadable via video description. The original audio was trimmed and converted from MP3 to OGG format.
+
+raccoon_squeak.ogg: audio is recorded by youtube user "jnargus" (https://youtu.be/BGjFP1CP7E0) licensed under Creative Commons Attribution license (reuse allowed)
+
+goat_bah.ogg: audio is provided by youtube user "Winry Marini" (https://youtu.be/QIhwzsk5bww) granted permission to be used in game
+
+lizard_happy.ogg: audio created by youtube user "Nagaty Studio" (https://youtu.be/I7CX0AS8RNI) licensed under Creative Commons Attribution license (reuse allowed).
+
```
Comment (juliangiebel) [inline Resources/Audio/Animals/license.txt:38]: I'm not to sure about the sounds from youtube where you just state that we can use those sounds.
For the sounds that have a licence you should state the proper licence name like above.
In this case that would be: CC BY 3.0

Comment (CrzyPotato) [reply Resources/Audio/Animals/license.txt:38]: Alright I will do that. I did however just to let you know asked them directly just in case.

**Review by juliangiebel — APPROVED** [review]
lgtm

**Comment by juliangiebel** [comment]
Good first PR 👍

**Comment by CrzyPotato** [comment]
Thank you

**Comment by CrzyPotato** [comment]
I accidentally committed fix for gibbed head being able to turn so I just decided to include it along with the other addition.

**Review by mirrorcult — APPROVED** [review]
good PR, in the future don't include separate changes like that in the same PR

**Pull request merged**